src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+
+const expensesService = require('../services/expenses.services');
+const userService = require('../services/users.services');
+const { expensesFilter } = require('../utils/expensesFilter');
+
+const getAll = (req, res) => {
+  const {
+    userId,
+    from,
+    to,
+    categories,
+  } = req.query;
+
+  const expenses = expensesFilter(userId, from, to, categories);
+
+  res.send(expenses);
+};
+
+const getById = (req, res) => {
+  const { id } = req.params;
+  const searcedExpense = expensesService.getById(+id);
```

Comment:
it is better to check whether id exists first, because if it is not it will be undefined and there might be some troubles with it

Reply:
Good point